scripts/tasks/ts.js

```diff
@@ -1,13 +1,13 @@
 module.exports = function(options) {
   options = options || {};
+  options.commonjsOnly = options.commonjsOnly || process.argv.indexOf('--commonJsOnly') > -1;
```

Comment:
Where is this option consumed? Is it built-in?

Reply:
further below in this file?

Reply:
Oh sorry, I thought it was a new option. 😄